chain/network/src/types.rs

```diff
@@ -1282,7 +1283,11 @@ pub enum PeerManagerRequest {
     UnregisterPeer,
 }
 
-pub struct EdgeList(pub Vec<Edge>);
+pub struct EdgeList(
```

Comment:
it is better to use named fields here. Otherwise it is hard to read `EdgeVerifier::handle`

Reply:
Fixed